scripts/deployment-utils.js

```diff
@@ -0,0 +1,25 @@
+const packages = [
+  { packageName: 'logs', service: 'browser-logs-sdk' },
+  { packageName: 'rum', service: 'browser-rum-sdk' },
+  { packageName: 'rum-slim', service: 'browser-rum-sdk' },
+]
+
+// ex: datadog-rum-v4.js
+const buildRootUploadPath = (packageName, version, extension = 'js') => `datadog-${packageName}-${version}.${extension}`
+
+// ex: us1/v4/datadog-rum.js
+const buildDatacenterUploadPath =
+  (datacenter) =>
+  (packageName, version, extension = 'js') =>
+    `${datacenter}/${version}/datadog-${packageName}.${extension}`
```

Comment:
Do you mean having something like `buildUploadPath(uploadPathType, packageName, version, extension)`?

Reply:
Yes

Reply:
With the current version, all the logic differences between root and datacenter are orchestrated by the main functions which makes quite explicit what happens in both cases.
With the suggested approach, it would move the logic to handle the difference of upload path to the deployment utils which would remove all root/datacenter concerns from the deploy script but the upload-sourcemaps script would still have some to handle.

For symmetry-sake between the two scripts, I'd be in favour of keeping the current version.

Reply:
Well that's subjective, but to me relying on currying and ternary makes it way harder than necessary. I don't see the issue with `upload-sourcemaps` having to more cases to handle for the root/datacenter distinction.

Reply:
wdyt of https://github.com/DataDog/browser-sdk/pull/2049/commits/c20718d71a151828342afabe25834470ae88d379?

Reply:
It looks better! 👍